src/components/customerCases/customerCase/CustomerCase.tsx

```diff
@@ -54,12 +63,14 @@ export default function CustomerCase({ customerCase }: CustomerCaseProps) {
                 {customerCase.projectInfo.delivery}
               </Text>
             </div>
-            <div className={styles.projectInfoItem}>
-              <Text>Konsulenter</Text>
-              <Text className={styles.projectInfoItemValue}>
-                {customerCase.projectInfo.consultants.join(", ")}
-              </Text>
-            </div>
+            {consultantsResult.ok && (
```

Comment:
isn't it enough to write consultantsResult?

Reply:
Are you thinking just `consultantsResult && ...`?

This is using the `Result` type, so `consultantsResult` will always be an object. So checking if `consultantsResult` is truthy would always return true. The proper check with `Result` is to check if `.ok` is true.